full-stack-tests/presentation/src/performance/GetElementProperties.test.ts

```diff
@@ -1,302 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
confirming we meant to delete this file in 4.11

Reply:
correct, we don't need these tests anymore